spec/header-files.md

```diff
@@ -21,28 +21,49 @@ A header file contains:
 
 Each line must be under 255 characters, and fields are separated by spaces or tabs (except where otherwise noted).
 
+Detailed documentation on header files can be found at: [https://physionet.org/physiotools/wag/header-5.htm](https://physionet.org/physiotools/wag/header-5.htm)
+
 ---
 
 ## Record Line
 
 The first non-comment line is the **record line**, which provides metadata about the overall record. It includes:
 
 | Field | Description |
-|:------|:------------|
+|:------|:-----------|
 | Record name | Identifier for the record (letters, digits, underscores only). |
 | Number of segments (optional) | If present, appended as `/n`. Indicates a multi-segment record. |
 | Number of signals | Number of signals described in the header. |
 | Sampling frequency (optional) | Samples per second per signal. Defaults to 250 if omitted. |
 | Counter frequency (optional) | Secondary clock frequency, separated from sampling frequency by a `/`. |
 | Base counter value (optional) | Offset value for counter, enclosed in parentheses. |
-| Number of samples (optional) | Total samples per signal. |
+| Number of samples (optional) | Total samples per signal when `samps_per_frame`=1; total frames otherwise. |
 | Base time (optional) | Start time of the recording (`HH:MM:SS`). |
 | Base date (optional) | Start date (`DD/MM/YYYY`). |
 
-**Example:**
+### Examples
+
+#### Basic header (uniform sampling frequency)(record 100 from the [MIT-BIH database](https://www.physionet.org/content/mitdb)):
+
 ```text
-100 2 360 650000 12:00:00 01/01/2000
+100 2 360 650000
 ```
+- `100`: Record name (must match the filename prefix).
```

Comment:
As I'm reading this, I'm trying to match the bullet to one of the fields in the table above. I think the bulleted list would be easier to follow if the descriptive text here exactly matched the "field" text above (e.g. instead of `Total samples for each signal.` we would put `Number of samples`). If necessary, we can provide additional information in parentheses.